src/main/java/org/opentripplanner/middleware/tripmonitor/jobs/CheckMonitoredTrip.java

```diff
@@ -416,16 +416,16 @@ private OtpGraphQLVariables getQueryParamsForTargetZonedDateTime() {
     }
 
     /**
-     * Get the latest tracked journey associated with this trip, if available. If rerouting has occurred as part of that
-     * tracked journey, use the last rerouting location as the 'from place' instead of the original attributed to the
-     * trip.
+     * Get the latest tracked journey associated with this trip, if available and live. If rerouting has occurred as
+     * part of that tracked journey, use the last rerouting location as the 'from place' instead of the original
+     * attributed to the trip.
      */
     public void checkForRerouting(OtpGraphQLVariables params) {
         TrackedJourney trackedJourney = Persistence.trackedJourneys.getOneFiltered(
             eq("tripId", trip.id),
             Sorts.descending("dateCreated")
         );
-        if (trackedJourney != null && trackedJourney.hasRerouted()) {
+        if (trackedJourney != null && !trackedJourney.hasEnded() && trackedJourney.hasRerouted()) {
```

Comment:
Class `CheckMonitoredTrip` has a method `isTrackingOngoing`, should we reuse that or replace it?

Reply:
I think we use the `isTrackingOngoing` method, but update `TripTrackingData.getOngoingTrackedJourney` to include `Sorts.descending("dateCreated")` so we are certain to be working with the latest tracked journey.